cms/static/js/views/overview.js

```diff
@@ -1,242 +0,0 @@
-define(["domReady", "jquery", "jquery.ui", "underscore", "gettext", "common/js/components/views/feedback_notification",
```

Comment:
As a result of this work? Or you just happened to notice that it wasn't used? If the latter, I think it makes sense to make a separate commit for this change.